src/main/java/uk/gov/hmcts/reform/sscs/personalisation/Personalisation.java

```diff
@@ -128,20 +116,47 @@
         personalisation.put(ONLINE_HEARING_REGISTER_LINK_LITERAL, config.getOnlineHearingLink() + "/register");
         personalisation.put(ONLINE_HEARING_SIGN_IN_LINK_LITERAL, config.getOnlineHearingLink() + "/sign-in");
 
+        personalisation.put(APPOINTEE_DESCRIPTION, getAppointeeDescription(subscriptionType, ccdResponse));
```

Comment:
This Appointee_Description is not a field in the template. Therefore, is not going to show up, isn't it?

Reply:
We added APPOINTEE_DESCRIPTION to the templates to allow us to reuse Appellant templates in an Appointee context

Reply:
That's why we updated certain template IDs in previous commits.